### ISSUE TITLE
About pre-trained models

### ISSUE THREAD

**Issue by tuokaikyle** [yiranran/QMUPD#4]
Hi, 

Just wondering, what is the difference between the pre-trained model in this repo and the one in Unpaired-Portrait-Drawing? They seem to have the same file and size.

Thanks

**Comment by yiranran** [comment]
The network structure is the same, but **the training strategy is different**. 
We use a new quality metric guided loss term for training this model. 
Please check the paper https://ieeexplore.ieee.org/document/9699090 for more details.
